tests/MockTest.php

```diff
@@ -64,8 +64,14 @@ public function testRecording()
         time();
         $this->assertEquals(array(array()), $recorder->getCalls());
         
-        time(true);
-        $this->assertEquals(array(array(), array(true)), $recorder->getCalls());
+        $mock = new Mock(__NAMESPACE__, "abs", 'emptyFunc');
+        $mock->enable();
+        $recorder = $mock->getRecorder();
+        $this->assertEmpty($recorder->getCalls());
+
+        abs(12);
+        $this->assertEquals(array(array(12)), $recorder->getCalls());
+
     }
```

Comment:
I'm not actually sure why I removed that second ```time``-test. Seemd logical at the time of doing so....

I assume it was due to the fact that the test always failed due to the fact that the parameter to time is not necessary and therefore not added to the recording-array. Therefore the assertion would have to be rewritten to ```$this->assertEquals(array(array(), array()), $recorder->getCalls());``` where I assumed that it is a simple repetition of the previous assertion. But it also tests that **both** calls to the function are present in the recorder which is no not tested any more.

So I'll add that one with the "corrected" assertion.